src/CMakeLists.txt

```diff
@@ -27,13 +27,21 @@ include(CMake/settings.cmake)
 # set to something different then a development component will be created with
 # link libraries and header which are then excluded from runtime component
 set (CMAKE_INSTALL_DEFAULT_COMPONENT_NAME "xrt")
-
+  
 # Enable development package by specifying development component name
 # If XRT_DEV_COMPONENT is same DEFAULT_COMPONENT then only that package
 # is created with both development and run-time content.
 #set (XRT_DEV_COMPONENT "xrt-dev")
+set (XRT_COMPONENT "xrt")
 set (XRT_DEV_COMPONENT "xrt")
 
+# For NPU builds the defalt component is "npu".
+if (XRT_NPU)
```

Comment:
This changes install package name to "xrt-npu". Please review whether we want to pursue it.

Reply:
Agreement that we will pursue -npu build switch with separate -npu.{deb,rpm} knowing that -npu and -xrt canot be installed side-by-side.